lib/auth/apiserver.go

```diff
@@ -758,6 +759,7 @@ func (s *APIServer) authenticateWebUser(auth ClientI, w http.ResponseWriter, r *
 		return nil, trace.Wrap(err)
 	}
 	req.Username = p.ByName("user")
+	req.ClientIP = strings.Split(r.RemoteAddr, ":")[0]
```

Comment:
Do you think we should add some checks here?
Hopefully we always have "IP:port" here, but better be safe than panic

Reply:
It wouldn't panic, splitting is always safe, it'd simply fail (with some access denied exception) down the road.
Either way, I've switched to [utils.Host](https://github.com/gravitational/teleport/blob/master/lib/utils/utils.go#L282) as @rosstimothy suggested